data/brands/tourism/hotel.json

```diff
@@ -18,6 +18,17 @@
     }
   },
   "items": [
+    {
+      "displayName": "Flair Hotels",
+      "locationSet": {"include": ["de"]},
+      "matchNames": ["Flair Hotels eV"],
+      "tags": {
+        "brand": "Flair Hotels",
+        "brand:wikidata": "Q126066863",
+        "name": "Flair Hotels",
```

Comment:
I'd do singular on the name, the website seems to mix `Flair Hotel` and `Flair Hotels` with the logo being `Flair Hotels` and the text usually being `Flair Hotel`.

Reply:
```suggestion
        "name": "Flair Hotel",
```